.github/CHANGELOG.md

```diff
@@ -83,6 +83,8 @@
 1. [A380X] Fix EWD avail. thrust fill area & PFD rudder trim visibility on ground - @flogross89 (floridude)
 1. [A380X/LIGHTS] Fix function of FCU brightness knobs - @heclak (Heclak)
 1. [A380X/FWS] Fix "NO ZFW OR ZFWCG DATA" ECAM alert after landing - @flogross89 (floridude)
+1. [GSX] Automatic Refuel start and Cargo Door Synch for A380, External Power Avail Synch with GSX for A380/A320 - @Fragtality (Fragtality)
```

Comment:
```suggestion
1. [GSX] Automatic Refuel start and Cargo Door Synch for A380X, External Power Avail Synch with GSX for A380X/A32NX - @Fragtality (Fragtality)
```

Reply:
It is not random, just mixed Grammar of two Languages and laziness :zany_face: